src/components/Resolution.tsx

```diff
@@ -772,6 +774,26 @@ export default class Resolution extends React.Component<Props, State> {
     }
   }
 
+  deleteResolution = () => {
+    return (<div>
```

Comment:
could you componentise this a la https://github.com/MaxwellBo/Muncoordinated-2/blob/513dac9fb2069a81912771e7b73e0c71dec893ab/src/components/Strawpoll.tsx#L91 so the `open` state is isolated to its own component?